satpy/etc/readers/viirs_l2_cloud_mask_nc.yaml

```diff
@@ -0,0 +1,50 @@
+reader:
+  name: viirs_l2_cloud_mask_nc
+  short_name: VIIRS CSPP Cloud Mask
+  long_name: VIIRS CSPP Cloud Mask data in NetCDF4 format
+  description: VIIRS CSPP Cloud Mask reader
+  status: alpha
```

Comment:
Beta maybe?

Reply:
Upgraded (not pushed yet).

Reply:
Done in 23b0a419a8fe58bec1e439081005caae15083105